### PR TITLE
Typo:  "image" should be "images" in for loop.  Error occurs otherwise.

### PR DESCRIPTION
**Pull request by jantic**
It looks like after this for loop was run successfully, the 's' in 'images' somehow got cut off and committed this way.

### CODE REVIEW

File: stable_diffusion.ipynb [hunk 1]
```diff
@@ -2632,7 +2633,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.10"
+   "version": "3.10.6"
```
Comment (pcuenca) [inline stable_diffusion.ipynb:2636]:
```suggestion
```

File: stable_diffusion.ipynb [hunk 2]
```diff
@@ -1497,6 +1497,7 @@
    "cell_type": "code",
    "execution_count": 14,
    "metadata": {
+    "collapsed": true,
```
Comment (pcuenca) [inline stable_diffusion.ipynb:1500]:
```suggestion
```

**Review by pcuenca — APPROVED** [review]
Awesome, thanks!

**Comment by pcuenca** [comment]
Oops there are conflicts now, could you please take a look? Otherwise I can do it tomorrow, no problem!

**Comment by jantic** [comment]
Crap, I don't have write access to do the "Resolve conflicts".  But it's literally just changing 'Image' to 'Images' in a for loop. so whatever is outside of that change can overwrite whatever else was introduced by me...

**Comment by pcuenca** [comment]
Cool, no problem, I'll try to fix it.

**Pull request merged**